src/components/SelectNetwork.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <dialog-base
+    :visible.sync="isVisible"
+    :title="t('bridge.selectNetwork')"
+    class="networks"
+    width="464px"
+  >
+    <p class="networks-info">{{ t('bridge.networkInfo') }}</p>
+    <s-radio
+      v-model="selectedNetwork"
+      v-for="network in subNetworks"
+      :key="network.id"
+      :label="network.name"
+      class="network"
+      @change="selectNetwork"
+    >
+      <span class="network-name">{{ t(`bridge.${network.name}`) }}</span>
+      <token-logo :tokenSymbol="network.symbol" />
+    </s-radio>
+  </dialog-base>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator'
+import { Getter, Action } from 'vuex-class'
+
+import TranslationMixin from '@/components/mixins/TranslationMixin'
+import DialogMixin from '@/components/mixins/DialogMixin'
+import DialogBase from '@/components/DialogBase.vue'
+import { Components, BridgeNetwork } from '@/consts'
+import { lazyComponent } from '@/router'
+
+@Component({
+  components: {
+    DialogBase,
+    TokenLogo: lazyComponent(Components.TokenLogo)
+  }
+})
+export default class SelectNetwork extends Mixins(TranslationMixin, DialogMixin) {
+  selectedNetwork = ''
+
+  @Getter('subNetworks', { namespace: 'web3' }) subNetworks!: Array<BridgeNetwork>
+  @Getter('ethNetwork', { namespace: 'web3' }) ethNetwork!: string
+  @Getter('evmNetwork', { namespace: 'web3' }) evmNetwork!: string
+
+  @Action('setEvmNetwork', { namespace: 'web3' }) setEvmNetwork
+
+  created () {
+    if (this.evmNetwork) {
+      this.selectedNetwork = this.evmNetwork
+    } else {
+      this.selectedNetwork = this.subNetworks?.find(item => item.name === this.ethNetwork) ? this.ethNetwork : this.subNetworks[0]?.name
```

Comment:
This `find `expression is correct? It compares different data, `sub network name` and  `network type` and always will be falsy